keyboards/kbd75/keymaps/kingwangwong/keymap.c

```diff
@@ -0,0 +1,148 @@
+#include QMK_KEYBOARD_H
+
+#define _QW 0
+#define _FO 1
+#define _FL 2
+#define _CL 3
+#define _DL 4
+
+enum planck_keycodes {
+  QWERTY = SAFE_RANGE,
+};
+
+// Helpful defines
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
+
+// Custom macros
+#define SPC_FUN LT(_FL, KC_SPC)
+#define BSC_CON LT(_CL, KC_BSPC)
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  /* 0: Qwerty layer
+   * ┌─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┬─────┐
+   * │ ESC │ F1  │ F2  │ F3  │ F4  │ F5  │ F6  │ F7  │ F8  │ F9  │ F10 │ F11 │ F12 │PRSCR│ INS │HOME │
+   * ├─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┼─────┤
+   * │  `  │  1  │  2  │  3  │  4  │  5  │  6  │  7  │  8  │  9  │  0  │  -  │  =  │BSLS │ DEL │ END │
+   * ├─────┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴─────┼─────┤
+   * │TAB     │  Q  │  W  │  E  │  R  │  T  │  Y  │  U  │  I  │  O  │  P  │  [  │  ]  │BSPC    │PGUP │
+   * ├────────┴┬────┴┬────┴┬────┴┬────┴┬────┴┬────┴┬────┴┬────┴┬────┴┬────┴┬────┴┬────┴────────┼─────┤
+   * │CAPS     │  A  │  S  │  D  │  F  │  G  │  H  │  J  │  K  │  L  │  ;  │  '  │ENTER        │PGDN │
+   * ├──────┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴──┬──┴───────┬─────┼─────┤
+   * │LSHFT │LSHFT│  Z  │  X  │  C  │  V  │  B  │  N  │  M  │  ,  │  .  │  /  │RSHFT     │ UP  │MO_CL│
+   * ├──────┼─────┴─┬───┴──┬──┴─────┴────┬┴─────┼─────┴─────┴────┬┴────┬┴────┬┴────┬─────┼─────┼─────┤
+   * │LCTRL │L_GUI  │L_ALT │SPC          │MO(_FL│BSPC            │R_ALT│R_GUI│R_CTR│LEFT │DOWN │RIGHT│
+   * └──────┴───────┴──────┴─────────────┴──────┴────────────────┴─────┴─────┴─────┴─────┴─────┴─────┘
+   */
+
+  /* 0: QWERTY */
+  [_QW] = LAYOUT(
+    KC_ESC,   KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,   KC_PSCR,  KC_INS,   KC_HOME, \
```

Comment:
The backslash escaping isn't necessary here, only for the layout `#define`.

Reply:
Ok thank you, I understand now that for all keymap.c files I don't need these backslashes, but I'm not sure what layout #define is

Reply:
Also I have decided I do not want to add my XD75 keymaps anymore since I don't have one anymore nor will I be updating it, how may I remove them?

Reply:
By that I mean the `#define LAYOUT(...) {...}` that is found in the keyboard (as opposed to keymap) code - here it would be kbd75.h, however I just noticed something about this. I'll put it in a proper comment so it's more visible.

To remove your keymaps you can just delete your keymap folder and do a `git add .` as usual. Then you can commit the deletions.

Reply:
I've moved my kbd75 and quefrency to the correct folders now. 
I have the rev2, I'm not too sure how I would implement that code, would I just write #include "rev2.h" in the keymap?
Thanks

Reply:
I think you just have to specify the revision when compiling eg:
`make kbdfans/kbd75/rev2:kingwangwong`